### PR TITLE
Update CNAME

### PR DESCRIPTION
**Pull request by beneshed**

### CODE REVIEW

File: CNAME [hunk 1]
```diff
@@ -1 +1 @@
-opensource.forter.com
\ No newline at end of file
+opensource.forter.dev
```
Comment (bennypowers) [inline CNAME:1]:
```suggestion
forter.dev
```

Comment (bennypowers) [reply CNAME:1]: why not this?

Comment (bennypowers) [reply CNAME:1]: then `blog.forter.dev` later

Comment (beneshed) [reply CNAME:1]: Github does not allow you to use a root DNS entry

Comment (bennypowers) [reply CNAME:1]: https://help.github.com/en/articles/setting-up-an-apex-domain ??

**Pull request closed without merging**